conf/defaults.ini

```diff
@@ -1357,3 +1357,13 @@ index_update_interval = 10s
 # Move a specific app plugin page (referenced by its `path` field) to a specific navigation section
 # Format: <Page URL> = <Section ID> <Sort Weight>
 [navigation.app_standalone_pages]
+
+
+#################################### Secure Socks5 Datasource Proxy #####################################
+[secure_socks_datasource_proxy]
```

Comment:
Just FYI I think it's still convention to add this same section to `conf/sample.ini`